docs/x509.rst

```diff
@@ -166,6 +166,155 @@ X.509 Certificate Object
+
+        :type: :class:`Name`
+
+        The :class:`Name` of the issuer.
+
+        :raises cryptography.x509.UnknownAttribute: If there is an unknown
+            attribute in the issuer.
+
+    .. attribute:: subject
+
+        .. versionadded:: 0.8
+
+        :type: :class:`Name`
+
+        The :class:`Name` of the subject.
+
+        :raises cryptography.x509.UnknownAttribute: If there is an unknown
+            attribute in the subject.
+
+
+.. class:: Name
+
+    .. versionadded:: 0.8
+
+    An X509 Name is an ordered list of attributes. The entire list can be
+    obtained with :attr:`attributes` or you can use the helper properties to
+    obtain the specific type you want.
+
+    .. attribute:: attributes
+
+        :type: list
+
+        A list of all the :class:`Attribute` objects.
+
+    .. attribute:: country_name
+
+        :type: list
+
+        A list of country name :class:`Attribute` objects.
+
+    .. attribute:: organization_name
+
+        :type: list
+
+        A list of organization name :class:`Attribute` objects.
+
+    .. attribute:: organizational_unit_name
+
+        :type: list
+
+        A list of organizational unit name :class:`Attribute` objects.
+
+    .. attribute:: dn_qualifier
+
+        :type: list
+
+        A list of DN qualifier :class:`Attribute` objects.
+
+    .. attribute:: state_or_province_name
+
+        :type: list
+
+        A list of state or province name :class:`Attribute` objects.
+
+    .. attribute:: common_name
+
+        :type: list
+
+        A list of common name :class:`Attribute` objects.
+
+    .. attribute:: serial_number
+
+        :type: list
+
+        A list of serial number :class:`Attribute` objects. This is not the
+        same as the certificate's serial number.
+
+    .. attribute:: locality_name
+
+        :type: list
+
+        A list of locality name :class:`Attribute` objects.
+
+    .. attribute:: title
+
+        :type: list
+
+        A list of title :class:`Attribute` objects.
+
+    .. attribute:: surname
+
+        :type: list
+
+        A list of surname :class:`Attribute` objects.
+
+    .. attribute:: given_name
+
+        :type: list
+
+        A list of given name :class:`Attribute` objects.
+
+    .. attribute:: pseudonym
+
+        :type: list
+
+        A list of pseudonym :class:`Attribute` objects.
+
+    .. attribute:: generation_qualifier
+
+        :type: list
+
+        A list of generation qualifier :class:`Attribute` objects.
+
+    .. attribute:: domain_component
+
+        :type: list
+
+        A list of domain component :class:`Attribute` objects.
+
+    .. attribute:: email_address
+
+        :type: list
+
+        A list of email address :class:`Attribute` objects.
+
+.. class:: Attribute
+
+    .. versionadded:: 0.8
+
+    .. attribute:: oid
+
+        :type: str
+
+        The attribute OID (e.g. ``2.5.4.3``)
+
+    .. attribute:: name
+
+        :type: str
+
+        The name of the OID (e.g. ``commonName``)
```

Comment:
I think you should use quotes both here and in `oid` to make it clear they're strings.